pages/Documents/ConversationalAI/Third-PartyBots/LivePersonFunctions/advanced-features.md

```diff
@@ -412,3 +412,60 @@ function lambda(input, callback) {
 We can see the above LivePerson function in action below:
 
 <img class="fancyimage" style="width:300px" src="img/faas/faas_richcontent_demo.gif">
+
+### Receiving Last consumer message (Messaging Only)
+
+Third-Party bot now provides a way to add the last consumer message as a part of [the welcome event](third-party-bots-liveperson-functions-basic-content.html#the-welcome-event) (messaging only). When an ongoing conversation gets transferred to a new Agent or Skill, This enhancement will allow brands to respond to the last consumer message uttered as per their needs.
```

Comment:
you can remove `(messaging only)` from the description as it is in the heading already